website/project/forms.py

```diff
@@ -17,3 +17,4 @@ class NewNodeForm(Form):
     ], widget=BootstrapTextInput())
     description = TextAreaField('Description', widget=BootstrapTextArea())
     category = TextAreaField('Category', widget=BootstrapTextArea())
+    inheritContributors = TextAreaField('Category', widget=BootstrapTextArea())
```

Comment:
I changed the last two to text fields.  Were you thinking I should create a drop down field and/or boolean or checkbox field, or are the text fields cool enough?